scala/scala.bzl

```diff
@@ -127,7 +129,17 @@ _common_attrs.update({
     # on the other hand any internal change (i.e. on code that ijar omits) WON’T trigger recompilation by transitive dependencies
     "_dependency_analyzer_plugin": attr.label(
         default = Label(
-            "@io_bazel_rules_scala//third_party/plugin/src/main:dependency_analyzer"
+            "@io_bazel_rules_scala//third_party/dependency_analyzer/src/main:dependency_analyzer"
+        ),
+        allow_files = [".jar"],
+        mandatory = False),
+    "unused_dependency_checker_mode": attr.string(
+        values = ["warn", "error", "off", ""],
```

Comment:
If this is not set then bazel handles that as an empty string. If that's not set as one of the possible values it fails